stats/requested_books.php

```diff
@@ -19,7 +19,7 @@
 echo "<p class='warning'>This page has been temporarily disabled due to its high load on the system. We're working to fix this.</p>";
 exit;
 
-echo "<h2>" . _("Most Requested Books Being Proofread") . "</h2>\n";
+echo "<h2>" . _("Most Requested Books Being Proofread") . "</h2>\n"; /** @phpstan-ignore-line The exit statement above makes this dead code */
```

Comment:
Err, but it also makes all of the rest of this dead code -- is phpstan OK with that?

Reply:
Nevermind, I saw the discussion in Slack.